### ISSUE TITLE
Homepage / Landing Page

### ISSUE THREAD

**Issue by nelsonic** [dwyl/github-backup#55]
As a `user` who is `new` to the GitHub Issue Backup App,
I `want` to learn more about the app
`so that` I can add it to my GitHub organisation

At present the homepage of the app/site is the original "hard-coded" data/UI: 
https://github-backup.herokuapp.com/
![image](https://user-images.githubusercontent.com/194400/37088605-dbcadd20-21f5-11e8-88b2-624c9702ad3a.png)

As the _first_ page some people will see, we need to have a nice-looking landing page.

## Todo

+ [ ] Chose a Landing Page "theme" or "template" (_some research required_) inspiration: https://unbounce.com/landing-page-templates
+ [ ] Clear and concise copy.
  + [ ] _Exactly_ what "problem" are we trying to solve?
     + [ ] GitHub being "Offline"
     + [ ] People editing issues and changing the "_scope_" without accountability.
  + [ ] Enumerate the steps required to setup: 
    + [ ] Enable the App
    + [ ] Select Organisation
    + [ ] Select repository to track
    + [ ] Edit a issue comment
    + [ ] Show/see the previous version on ["Gibu"](#13)
  + [ ] "***Free***" if you give us "beta feedback"!
+ [ ] Screenshots where applicable. https://en.wikipedia.org/wiki/A_picture_is_worth_a_thousand_words
+ [ ] Mini (GIF?) video of enabling the app and then it tracking issue comment edits?
+ [ ] Enable **Google Analytics** on the homepage: https://github.com/dwyl/github-backup/issues/54

Notes:
This is not a priority _until_ the remaining MVP _features_ are complete. 
However, once the core functionality is in place we should get this done "ASAP"
`so that` we can get _feedback_ from the community without it looking too "unfinished".
